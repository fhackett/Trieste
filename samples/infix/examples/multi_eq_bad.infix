x = 5 = 6;
x = 1 =; // this is not an error, surprisingly
x = = 1;