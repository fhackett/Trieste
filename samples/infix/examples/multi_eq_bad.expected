//! expect_parse_fail
//! --enable-tuples expect_parse_fail
//! --enable-tuples --tuples-require-parens expect_parse_fail
//! --enable-tuples --tuples-require-parens --use-parser-tuples expect_parse_fail
Invalid assign
-- multi_eq_bad.infix:0:0
x = 5 = 6;
~~~~~~~~~

Invalid assign
-- multi_eq_bad.infix:2:0
x = = 1;
~~~~~~~
